### ISSUE TITLE
The `getContent` method references the method `getXmlElement` which does not appear to exist.

### ISSUE THREAD

**Issue by chanced** [yjs/y-websocket#86]
**Checklist**

* [X] Are you reporting a bug? Use github issues for bug reports and feature requests. For general questions, please use https://discuss.yjs.dev/

**Describe the bug**

the function `getContent` references `getXmlElement` on a `Y.Doc` which does not appear to exist.

https://github.com/yjs/y-websocket/blob/caa6304ba30c0e6d5921e735470f6470b045cb6a/bin/callback.js#L73

https://github.com/yjs/yjs/blob/dc9717ecd0650315d87129cff795dc5116f8bcf2/src/utils/Doc.js

**To Reproduce** I'm not sure that code branch can ever be reached but I wanted to let you know regardless.

**Environment Information**

* y.js 13.5.6 per y-websocket's dependency
* y-websocket - latest

<sub><a href="https://huly.app/guest/w-githubdmonad-yjs-660d5772-70f06a0a22-d28cf5?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjBkNjQxYzhjN2YzNTBiMTEyMTQwZmQiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InctZ2l0aHViZG1vbmFkLXlqcy02NjBkNTc3Mi03MGYwNmEwYTIyLWQyOGNmNSIsInByb2R1Y3RJZCI6IiJ9.eT8rCNcZdF2cVKu3FsgJPqQtjMxmw92rlg8yDf3owvk">Huly&reg;: <b>YJS-512</b></a></sub>

**Comment by dmonad** [comment]
Right, thanks! We should adopt this to `ydoc.get(objName, Y.XmlElement)`.
